### PR TITLE
Add logging to debug geolocation updates

### PR DESCRIPTION
**Pull request by NeonDaniel**
Use `self.location` param to avoid repetitive service calls
Emit error responses for calling modules to handle

### CODE REVIEW

File: ovos_phal_plugin_ipgeo/__init__.py [hunk 1]
```diff
@@ -2,13 +2,15 @@
...
+from ovos_utils.log import LOG
+from urllib3.exceptions import NewConnectionError
 
 
 class IPGeoPlugin(PHALPlugin):
     def __init__(self, bus=None, config=None):
         super().__init__(bus, "ovos-phal-plugin-ipgeo", config)
-        self.location = {}
         self.web_config = LocalConf(get_webcache_location())
+        LOG.debug(f'web_config has keys: {self.web_config.keys()}')
```
Comment (JarbasAl) [inline ovos_phal_plugin_ipgeo/__init__.py:13]: isnt this a bit spammy and excessive? we dont care about this info....

if anything the useful log would be about location being present or not only, thats what we care about, whoever is reading the logs will not know why the keys are logged or what to make of it

also perhaps this should only be logged on actual update?

File: ovos_phal_plugin_ipgeo/__init__.py [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
 from ovos_utils.configuration import get_webcache_location, LocalConf
 from ovos_utils.messagebus import Message
+from ovos_utils.log import LOG
+from urllib3.exceptions import NewConnectionError
 
 
 class IPGeoPlugin(PHALPlugin):
     def __init__(self, bus=None, config=None):
         super().__init__(bus, "ovos-phal-plugin-ipgeo", config)
-        self.location = {}
```
Comment (JarbasAl) [inline ovos_phal_plugin_ipgeo/__init__.py:10]: this was meant to save repeat http calls, any reason to not store the value?

File: ovos_phal_plugin_ipgeo/__init__.py (hunk 1 above)
Comment (NeonDaniel) [reply ovos_phal_plugin_ipgeo/__init__.py:13]: Yes, shouldn't have been committed this was my testing a separate issue

File: ovos_phal_plugin_ipgeo/__init__.py (hunk 2 above)
Comment (NeonDaniel) [reply ovos_phal_plugin_ipgeo/__init__.py:10]: I suppose not; any time the plugin is reloaded we'll get a fresh location, I think the only edge case would be updating on network change but that should be handled explicitly anyways

**Pull request merged**